h2o-helm/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+name: h2o
+description: H2O open source machine learning platform official Helm chart
+
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 1.0.0
```

Comment:
Just a question - should be this numbers separated to our version numbers?

Reply:
The version of the Helm chart could be aligned with H2O release. Good point. cc @michalkurka .

Would be great to see it start with 3.32.0.1, but why not ?

Reply:
yes, I agree with @satai - would be good to keep those 2 in synch - the versions should be the sae

Reply:
We'll only have one then.

Reply:
yeah, that is what we proposed internally here: https://h2oai.slack.com/archives/C0BGE8ZJP/p1600973747002400?thread_ts=1600970951.001100&cid=C0BGE8ZJP

if you think this approach has issue - we should revisit the discussion

Reply:
I don't. It's fine.